go.mod

```diff
@@ -58,7 +57,7 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200331152225-585af27e34fd // release-4.5
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
 
-	github.com/operator-framework/api => github.com/operator-framework/api v0.3.1
+	github.com/operator-framework/api => github.com/operator-framework/api v0.3.2
```

Comment:
Why does `github.com/operator-framework/api` need a replaces directive instead of bumping the required version to v0.3.2 directly?

Reply:
Sounds good - addressed in the latest commit.